### PR TITLE
Fix problem when --devel mod is used

### PR DESCRIPTION
**Pull request by j-mracek**
It renames feature file after test to original name to not rename local project
files

### CODE REVIEW

**Comment by kkaarreell** [comment]
The behave doesn't have to _crash_, it would simply exit with non-zero exit code due to a failing test.

**Comment by j-mracek** [comment]
@ignatenkobrain Refactored. Please can you make very fast review. It is very important.

File: Dockerfile.local [hunk 1]
```diff
@@ -3,6 +3,7 @@ ENV LANG C
 
 COPY dnf-docker-test/repo /var/www/html/repo/
 COPY dnf-docker-test/features /behave/
+COPY dnf-docker-test/features /tests/
```
Comment (ignatenkobrain) [inline Dockerfile.local]: what's the point to copy it 2 times?

File: dnf-docker-test/launch-test [hunk 2]
```diff
@@ -3,5 +3,10 @@ set -xeuo pipefail
 
 httpd -k start
 new_name=${1}-${2}
-mv /behave/${1}.feature /behave/${new_name}.feature
+EXIT_STATUS=0
+cp /tests/${1}.feature /behave/${new_name}.feature || EXIT_STATUS=1
```
Comment (ignatenkobrain) [inline dnf-docker-test/launch-test]: if you want to exit here, just don't do `||`, it doesn't make any sense

File: Dockerfile.local (hunk 1 above)
Comment (j-mracek) [inline Dockerfile.local]: ```COPY dnf-docker-test/features /behave/```
Here I need to copy everything except *.feature files (dirs and .*py files)
```COPY dnf-docker-test/features /tests/```
Here I need to copy *.feature files

File: dnf-docker-test/launch-test (hunk 2 above)
Comment (j-mracek) [inline dnf-docker-test/launch-test]: I would like to do two things:
1. Print human understable what went wrong("Feature file ${1}.feature not found") 
2. End return with exit code 1.

File: Dockerfile.local (hunk 1 above)
Comment (j-mracek) [inline Dockerfile.local]: Anyway, I see another problem in implementation. Steps are not shared.

**Comment by ignatenkobrain** [comment]
don't really like idea, but since we have to make some fix ASAP, merging.

**Comment by kkaarreell** [comment]
What is the purpose of renaming the feature file anyway? I can imagine that the only reason could be to let user easily find which dnf version was executed but given that the launch-test script prints every command this information is already available.

**Comment by j-mracek** [comment]
@kkaarreell The reason was simple. Tests create .xml output where name is taken from name of behave test. When we want to run dnf with python2 or puthon3, the second test overwrites the xml output from previous.  The reason was to enable test runs with multiple prefixes (dnf-2, dnf-3).

**Comment by kkaarreell** [comment]
I see. Thank you.

**Pull request merged**